scrapers/SpankBang.yml

```diff
@@ -11,11 +11,18 @@ xPathScrapers:
       Details: //meta[@name='description']/@content
       Image: //div[@class='play_cover']/img/@src
       Studio:
-        Name: //li[@class='us']/a[@class='ul'][contains(@href,'channel')]/text()
+        Name: //li[@class='us']/a[@class='ul']/text()
```

Comment:
The channel was there to make sure no  plain `users`  where added as studios.
We were considering studios only channels(<https://spankbang.com/channels>)
I dont use the scraper and i think sure most users would scrape from the original sources instead so i dont have an issue with this change if that is the intended usage?

Reply:
At least in my case it's the original creators posting there, so the "user" is the studio.
Not sure if this will create issues for others though.

Reply:
@bnkai  Spankbang is one of the homes of the PMV community, in this case it is the original source.
So getting as much data as possible from them will definitely be useful 😊